src/components/Input/InputAddon.tsx

```diff
@@ -79,6 +79,7 @@ export const InputAddon = ({
         <Select
           isDisabled={isDisabled}
           isFullWidth={false}
+          popupMatchSelectWidth={false}
```

Comment:
can you add to the boolean props the is, has or can prefix?

Reply:
Is a bool or number props, not only boolean. Is it needed also in this cases the prefix? Could be maybe misleading?

Reply:
If needed, it is ok also should? So, `shouldPopupMatchSelectWidth` would be ok?

Reply:
mmm so you can use a static width or a general true width. 
In this case I think the name may be fine as you propose. 
Are you sure it works? Because I remember the width is set by the css as “unset” to avoid automatic inheritance. if worts for me is approved